Sources/Apodini/Properties/Binding.swift

```diff
@@ -32,10 +32,12 @@ private enum Retrieval<Value> {
 /// or an `@Environment`. The latter `Binding`s for the latter two are contained in their
 /// `projectedValue`s.
 @propertyWrapper
-public struct Binding<Value>: DynamicProperty, PotentiallyParameterIdentifyingBinding {
+public struct Binding<Value: Codable>: DynamicProperty, PotentiallyParameterIdentifyingBinding {
     private let store: Properties
     private var retrieval: Retrieval<Value>
-    fileprivate let parameterId: UUID?
+    let parameterId: UUID?
+    let parameterOptions: PropertyOptionSet<ParameterOptionNameSpace>
```

Comment:
I dislike how those properties are stored redundantly. There should be a single source of truth. Please modify the `Retrieval` enum to differentiate between `Environment` and `Parameter`, so you can obtain the `Parameter` and its required properties from there.

Reply:
This is a good point, I can just use the `Parameter` stored in the store. I will change the implementation to reflect the suggestion